franklin/builder/models.py

```diff
@@ -18,6 +20,24 @@ class Site(models.Model):
     url = models.CharField(max_length=100, default='', db_index=True)
     path = models.CharField(max_length=100)
 
+    def build(self, git_hash, repo_owner, path, repo_name):
+        url = os.environ['BUILDER_URL']
```

Comment:
Since all of these variables are related to a single model, would it make sense to just have this as a model method? `mysite.save()` then `mysite.build()`